src/Components/Nav/Nav.js

```diff
@@ -1 +1,248 @@
 import React from "react";
+import { useState, useEffect } from "react";
+import { superfluidNav } from "../../config";
+import styled from "styled-components";
+
+function Nav() {
+  const [shownSuperLogo, setShownSuperLogo] = useState(false);
+  const [shownSocial, setShownSocial] = useState(true);
```

Comment:
1. 두 상태값이 서로 연동되는 내용이라면 다음과 같이 쓸 수도 있겠습니다
```suggestion
  const [isSuperOrSocial, setIsSuperOrSocial] = useState({ super: false, social: true});
```
2. 아니면 아예 boolean 값 하나로만 해서 true면 superlogo가, false면 social이 보이게 할 수도 있겠습니다

Reply:
천재쓰 😲